spec-main/api-menu-spec.ts

```diff
@@ -882,7 +882,12 @@ describe('Menu module', function () {
       expect(output).to.include('Window has no menu')
     })
 
-    ifit(process.platform !== 'darwin')('does not override null menu on startup', async () => {
+    ifit(process.platform !== 'darwin')('does not override null menu on startup', async function () {
+      // This test is flaky on WOA.
```

Comment:
this is surprising to me. do we have any theories as to why?

Reply:
It is because the child process used for testing crashed on exit. Hmm I think I have just got an idea of how to work around this, will do it in another PR.